charts/helmet/templates/_deployment.yaml

```
@@ -36,6 +36,9 @@ spec:
       {{- if .Values.initContainers }}
       initContainers: {{- include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) | nindent 8 }}
       {{- end }}
+      {{- if .Values.podRestartPolicy }}
+      restartPolicy: {{- toYaml .Values.podRestartPolicy | nindent 8 }}
```

The value of the `restartPolicy` is an inline string such as Always, OnFailure, and Never. So you don't use the `toYaml` and `nindent`. Something like this:
```suggestion
      restartPolicy: {{ .Values.podRestartPolicy }}
```